sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -1503,7 +1529,11 @@ SUBROUTINE MDLFLD
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
+                 IF(P1D(I,J)<spval.and.T1D(I,J)<spval.and.Q(I,J,LL)<spval)THEN
```

Comment:
Should EGRID3(I,J) be checked in line 1532?

Reply:
will do

Reply:
done